processor/k8sattributesprocessor/README.md

```diff
@@ -474,51 +471,3 @@ as tags.
 
 By default, the `k8s.pod.start_time` uses [Time.MarshalText()](https://pkg.go.dev/time#Time.MarshalText) to format the
 timestamp value as an RFC3339 compliant timestamp.
-
-## Feature Gate
-
-### `k8sattr.fieldExtractConfigRegex.disallow`
```

Comment:
Instead of removing this completely, lets keep it around and document it as `stable` and no longe settable as of <version>. This will help users how are going through collector upgrades and will ensure any links saved to this heading still work.

Reply:
done